src/comment.cpp

```diff
@@ -304,7 +304,7 @@ void CommentParser::collectParameters(const char *fulltext, FileModule *root_mod
 
 		// Look for the group to which the given assignment belong
 		int i=0;
-		for (;i<groupList.size() && groupList[i].lineNo<firstLine;i++);
+		for (; (unsigned)i < groupList.size() && groupList[i].lineNo<firstLine;i++);
```

Comment:
Following code passed all the tests related to customzier:

`#include <boost/range/adaptor/reversed.hpp>`

``` 
for (const auto &groupInfo :boost::adaptors::reverse(groupList)){
            if (groupInfo.lineNo < firstLine) {
            //creating node for description
            shared_ptr<Expression> expr(new Literal(ValuePtr(groupInfo.commentString)));
            annotationList->push_back(Annotation("Group", expr));
            break;
        }
    }
```